src/Core/Framework/Test/Util/MemorySizeCalculatorTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
```

Comment:
wrong codestyle, the declare statement should also be in the first line.

running `composer run ecs-fix -- src` should fix this ;)